components/navbar.js

```diff
@@ -21,84 +22,165 @@ export default function HomeNavBar() {
-        <NavbarItem>
-          <Button 
-                as={Link}
-                color="default" 
-                variant="light"
-                size="lg"
-                href="https://isabeltovalles.com/cs-resources/spill-tea"
+          </NavbarMenuItem>
+          <NavbarMenuItem>
+            <Button 
+              as={Link}
+              color="default" 
+              variant="light"
+              size="lg"
+              href="https://isabeltovalles.com/contact-me"
+            >
+              Contact
+            </Button>
+          </NavbarMenuItem>
+          <NavbarMenuItem>
+            <Button 
+              isIconOnly 
+              onClick={handleThemeSwitch} 
+              color="default" 
+              variant="light"
+              size="md"
+              className="hover:rotate-180 transform origin-center"
+            >
+              {theme === 'light' ? '🌕' : '🌑'}
+            </Button>
+          </NavbarMenuItem>
+        </NavbarMenu>
+
+        {/* Menu items for larger screens */}
+        <NavbarContent justify="end" className="hidden sm:flex">
+          <NavbarItem>
+            <Button 
+              as={Link}
+              color="default" 
+              variant="light"
+              size="lg"
+              href="https://isabeltovalles.com/exp"
             >
-                Resources  
+              Experience
             </Button>
-        </NavbarItem>
-        <NavbarItem>
-          <Button 
-                as={Link}
-                color="default" 
-                variant="light"
-                size="lg"
-                href="https://isabeltovalles.com/socials"
+          </NavbarItem>
+          <NavbarItem>
+            <Button 
+              as={Link}
+              color="default" 
+              variant="light"
+              size="lg"
+              href="https://isabeltovalles.com/cs-resources/spill-tea"
             >
-                Socials
+              Resources  
             </Button>
-        </NavbarItem>
-        <NavbarItem>
-          <Button 
-                as={Link}
-                color="default" 
-                variant="light"
-                size="lg"
-                href="https://isabeltovalles.com/contact-me"
+          </NavbarItem>
+          <NavbarItem>
+            <Button 
+              as={Link}
+              color="default" 
+              variant="light"
+              size="lg"
+              href="https://isabeltovalles.com/socials"
             >
-                Contact
+              Socials
             </Button>
-        </NavbarItem>
-        <NavbarItem>
+          </NavbarItem>
+          <NavbarItem>
             <Button 
-                isIconOnly 
-                onClick={handleThemeSwitch} 
-                color="default" 
-                variant="light"
-                size="md"
-                className="hover:translate-x-1 hover:-translate-y-1 transition-transform duration-300 ease-in-out"
```

Comment:
again, would it be possible to create specific classes for these button styling so you don't have to add all these attributes over and over again?

Reply:
its just how tailwind css works